### ISSUE TITLE
injectived not built by `go install ...`

### ISSUE THREAD

**Issue by conorpp** [InjectiveLabs/sdk-go#146]
The readme instructs to `go install ...` and then use `injectived`.  However when I try to install I get a bunch of random looking tool s prefixed with a number, and none are `injectived`.

**Comment by achilleas-kal** [comment]
There's no need to install injectived to use the SDK - we'll modify the instructions. Could you reach out to me in a DM to help you out with the integration @conorpp ?

Telegram: achilleas_k
Discord: achilleaskalantzis

**Comment by achilleas-kal** [comment]
yes, you can install injectived here: https://github.com/OpenDeFiFoundation/injective-core
